packages/gatsby-theme-newrelic/src/components/FlipCard.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import Surface from './Surface';
+
+const FlipCard = ({ className, flipToBack, children }) => {
+  return (
+    <div
+      css={css`
+        width: 300px;
+        height: 300px;
+        background-color: transparent;
+        perspective: 1000px;
+      `}
+      className={className}
+    >
+      <div
+        css={css`
+          position: relative;
+          width: 100%;
+          height: 100%;
+          transition: transform 0.6s;
+          transform-style: preserve-3d;
+          ${flipToBack &&
+          css`
+            transform: rotateY(180deg);
+          `}
+        `}
+      >
+        {children}
+      </div>
+    </div>
+  );
+};
+
+const CardFront = ({ children, className }) => (
+  <Surface
+    base={Surface.BASE.PRIMARY}
+    css={css`
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      -webkit-backface-visibility: hidden;
+      backface-visibility: hidden;
+    `}
+    className={className}
+  >
+    {children}
+  </Surface>
+);
+
+const CardBack = ({ children, className }) => (
+  <Surface
+    base={Surface.BASE.PRIMARY}
+    css={css`
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      -webkit-backface-visibility: hidden;
+      backface-visibility: hidden;
+      transform: rotateY(180deg);
+    `}
+    className={className}
+  >
+    {children}
+  </Surface>
+);
+
+FlipCard.Front = CardFront;
+FlipCard.Back = CardBack;
+
+FlipCard.propTypes = {
+  className: PropTypes.string,
+  flipToBack: PropTypes.bool,
```

Comment:
Super nit-pick, could we name this variable something like `flipped`?  `flipToBack` feels like a state update vs the state of the component itself.

(i'm still gonna approve)

Reply:
ooh yes i was trying to think of the right term